### ISSUE TITLE
Keyboard navigation does not work for projected NgbDropdownItems

### ISSUE THREAD

**Issue by sorinfratila** [ng-bootstrap/ng-bootstrap#4381]
Please remember, the issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
Please read https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.

### Bug description:
I am have created a wrapper component around the dropdown component and I am projecting the NgbDropdownItems through an ng-content tag. The main functionality works, but I am unable to use keyboard navigation when the dropdown is open and also the dropdown does not close if I select an item through Enter key, although the click handler is fired.

Is there a way to get this functionality working with this setup ?

### Link to minimally-working StackBlitz that reproduces the issue:

[stackblitz](https://stackblitz.com/edit/angular-8pyuc5?file=src%2Fapp%2Fdropdown-basic.html,src%2Findex.html,src%2Fapp%2Fdropdown%2Fdropdown.component.ts,src%2Fapp%2Fdropdown%2Fdropdown.component.html,src%2Fapp%2Fdropdown%2Fdropdown.module.ts,src%2Fapp%2Fdropdown-basic.module.ts,src%2Fapp%2Fdropdown-basic.ts)

### Versions of Angular, ng-bootstrap and Bootstrap:

On stackblitz:

Angular: 14

ng-bootstrap: 13

Bootstrap: the stackblitz didn't require this.

On my project:
- I have lower versions, but it seems the issue is still there.
Angular: 12
ng-bootstrap: 9.1.2

**Comment by maxokorokov** [comment]
Note: doing `this.nativeElement.querySelectorAll('[ngbDropdownItem]'` will bring the correct elements, maybe we should use it instead.
